integration-tests/virtual-threads/vertx-event-bus-virtual-threads/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+quarkus.native.additional-build-args=--enable-preview
```

Comment:
Out of curiosity, what does this config property do?

Reply:
Until we get Java 21 support in native, we need to enable access to the virtual thread API explicitly. 
Note that the funny part is that when using Java 21, using this flag will fail the build :-D